example/home/models.py

```diff
@@ -36,6 +42,11 @@
 )
 from grapple.utils import resolve_paginated_queryset
 
+try:
```

Comment:
nitpic: I'd prefer a more explicit conditional usage in the panel definitions (i.e. `FieldPanel() if WAGTAIL_VERSION >= (3, 0) else StreamFieldPanel`